README.md

```diff
@@ -0,0 +1,219 @@
+# binge - version-less JavaScript package management for monorepos
+Binge is a version-less JavaScript package management tool for monolithic repositories (*monorepos*).
+
+It helps you to:
+* encourage modularization for a clearer separation of concerns,
+* enforce dependency version consistency across different local packages,
+* improve the developer experience and reduces the build file size by ensuring there is only one version of a dependency across all packages in the repository.
+
+[//]: # (To learn more about *why exactly* you should consider using binge, read our [announcement blog post]https://tech.signavio.com/2017/package-management-binge.)
+
+## Getting started with binge
+Install binge by running ``yarn global add binge``.
+
+**Note**: binge requires [Yarn](https://yarnpkg.com/lang/en/) and doesn't support npm.
+
+### Repository structure
+Structure your repository as follows:
+
+```
+…
+- client
+    - package1
+        - …
+        - src
+        - .babelrc
+        - .npmignore
+        - package.json
+        - yarn.lock
+    - …
+    - packageN
+        - …
+    - app
+        - src
+        - package.json
+        - …    
+…
+```
+
+Your client folder needs to contain a root-level folder for each local package, plus one additional folder that contains the global entry point to (the *root* of) your app.
+In our example (see: [./examples/watch-app](./examples/watch-app)), we name this folder ``app``.
+
+**Note:** binge supports nested local packages. However, we recommend you don't nest your packages.
+This reduces complexity and allow a clearer overview.
+
+Local packages have the same structure as ordinary npm packages, except that they are **not versioned**. 
+
+The ``package.json`` in the root folder lists the local packages as dependencies:
+
```

Comment:
Rewrite the above sentence to:
Local packages have the same structure as ordinary npm packages, except that they are not **versioned**. To include a reference to a local-package include it in the package.json as a "file reference"